javaagent-tooling/javaagent-tooling.gradle

```diff
@@ -37,6 +44,8 @@ dependencies {
   implementation deps.autoservice
   implementation deps.slf4j
 
+  implementation group: 'io.grpc', name: 'grpc-netty', version: '1.34.1'
```

Comment:
Not using `shaded`. Not bothering with bom for now since it's just one project

Reply:
Why not using `shaded`?

Reply:
Sorry ended up merging another PR into this one since I deleted everything here :)

https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2074

Adds 3MB (10%) to the agent jar with questionable benefit on latest Java 8.